components/toggle/Toggle.tsx

```diff
@@ -0,0 +1,51 @@
+import styles from "./Toggle.module.css";
+
+type ToggleProps = {
+  showMap: boolean;
+  setShowMap: Function;
+};
+
+const Toggle: React.FC<ToggleProps> = ({ showMap, setShowMap }) => {
+
+  return (
+    <div className={styles.toggleContainer}>
+      <label className={styles.toggleLabel}>
+        <input
+          value="List"
+          name="mode"
+          type="radio"
+          className={styles.toggleInput}
+          checked = {!showMap}
+        />
+        <span
+          className={styles.toggleText}
+          onClick={() => {
+            setShowMap(false);
+          }}
+        >
+          List
```

Comment:
Also List (here) and Map (in the other toggle) cannot be passed as props, so we could pass those as props to make this component reusable when needed